sql/load_demo_data.sql

```diff
@@ -6100,7 +6291,11 @@ CREATE TABLE `oh_session_audit` (
   `SEA_ACTIVE` tinyint(1) NOT NULL DEFAULT 1,
   PRIMARY KEY (`SEA_ID`),
   KEY `SEA_US_ID_A` (`SEA_US_ID_A`),
```

Comment:
In the multiple step version this key is <pre>`FK_SESSION_AUDIT_USER`</pre> not the value specified here.

Reply:
Sorry, I can see same `FK_SESSION_AUDIT_USER` name in both demo data and step_95 that introduced the FK.

Reply:
not exactly... both are `KEY` (`CONSTRAINT` is for FK), only change the name, because the script_95 dropped the _unnamed_ one in multiple scripts (step_84 was not defining it so the server assigned `OH_SESSION_AUDIT_IBFK_1`) and added again as `FK_SESSION_AUDIT_USER`.

I think we can fix demo data with the automatic name `OH_SESSION_AUDIT_IBFK_1` and in OP-1107 move the renaming command of the script_95 to a new step_XX, because it is actually a change and not a missing FK.

```
ALTER TABLE OH_SESSION_AUDIT
DROP FOREIGN KEY OH_SESSION_AUDIT_IBFK_1;

ALTER TABLE OH_SESSION_AUDIT
ADD CONSTRAINT FK_SESSION_AUDIT_USER
    FOREIGN KEY (SEA_US_ID_A)
    REFERENCES OH_USER (US_ID_A)
    ON DELETE CASCADE
    ON UPDATE CASCADE;
```

of course step_XX should be applied to demo data too.
Thanks!

Reply:
sorry I mispelled... step_84 didn't assign names to both KEY and CONSTRAINT, but anyway the two are connected each other.